prometheus_hardware_exporter/__main__.py

```diff
@@ -32,6 +40,24 @@ def parse_command_line() -> argparse.Namespace:
         default=10000,
         type=int,
     )
+    parser.add_argument(
+        "--redfish_host",
```

Comment:
**nitpick:** 

`redfish-host`?

Reply:
I changed it to `redfish-host` (and also similar for username and password)